tomcat/src/main/java/org/apache/catalina/connector/Connector.java

```diff
@@ -65,7 +74,7 @@ private void process(Socket connection) {
         }
         log.info("connect host: {}, port: {}", connection.getInetAddress(), connection.getPort());
         var processor = new Http11Processor(connection);
-        new Thread(processor).start();
+        threadPool.submit(() -> processor.process(connection));
```

Comment:
```suggestion
        threadPool.submit(processor);
```

Http11Processor 클래스의 run 메서드를 참고하세요.